package.json

```diff
@@ -29,6 +29,9 @@
     "lint:js": "eslint --cache .",
     "lint:types": "tsc --pretty --noEmit",
     "lint": "npm-run-all -l -p \"lint:**\"",
+    "fix:prettier": "npm run lint:prettier -- --write",
```

Comment:
I think `fmt` is better name, in other languages you use `fmt` or `format` command, for example:
- `rust` https://github.com/rust-lang/rustfmt
- `dart` https://dart.dev/tools/dart-format
- and etc, you can google it :smile:

Reply:
Yep, It's better 👍🏻